src/cli/cli.go

```diff
@@ -100,11 +83,36 @@ COPYRIGHT:
 	ErrJSONMarshal = errors.New("json marshal failed")
 )
 
-// App Wraps the app so that main package won't use the raw App directly,
-// which will cause import issue
-type App struct {
-	gcli.App
-}
+var (
+	cliConfig Config
+	apiClient *api.Client
+	quitChan  = make(chan struct{})
+)
+
+// flags used in cli commands
+var (
+	walletFile        string
+	address           string
+	changeAddress     string
+	many              string
+	password          string
+	jsonOutput        bool
+	csvFile           string
+	numAddresses      int
+	fiberNumAddresses int
+	coinName          string
+	label             string
+	useHex            bool
+	seed              string
+	strictSeed        bool
+	entropy           int
+	hideSecrets       bool
+	mode              string
+	encrypt           bool
+	addrsFilename     string
+	seedsFilename     string
+	overwrite         bool
+)
```

Comment:
defined all flags globally since they could be easily reused without affecting functionality of other commands

Reply:
Is it possible to scope them to their command?